modules/backend/formwidgets/FileUpload.php

```diff
@@ -159,6 +165,7 @@ protected function prepareVars()
         $this->vars['cssDimensions'] = $this->getCssDimensions();
         $this->vars['cssBlockDimensions'] = $this->getCssDimensions('block');
         $this->vars['useCaption'] = $this->useCaption;
+        $this->vars['maxFiles'] = $this->maxFiles;
```

Comment:
@RomainMazB we'll need to make sure this is an integer (or null)
```suggestion
        $this->vars['maxFiles'] = (is_int($this->maxFiles)) ? $this->maxFiles : null;
```

Reply:
I agree with you, that's better but why only maxFiles and not on maxFilesize, imageHeight or imageWidth?

Reply:
@RomainMazB I only commented on your actual changes - if you want to add it on the other properties too, go right ahead :)

Reply:
Done, and added to all the numeric options :+1: